local_modules/telemetry/index.js

```diff
@@ -26,19 +26,24 @@ module.exports = {
     'poi_backtolist',
     'poi_restore',
     'poi_share',
+    'poi_share_to',
     /* OSM */
     'poi_osm_open',
     'poi_osm_go',
     'poi_osm_favorite', // Favorite toggle
     'poi_osm_phone',
     'poi_osm_website',
+    'poi_osm_itinerary',
+    'poi_osm_osm_view',
+    'poi_osm_osm_edit',
     /* Pages Jaunes Poi */
     'poi_pages_jaunes_open',
     'poi_pages_jaunes_go',
     'poi_pages_jaunes_favorite', // Favorite toggle
     'poi_pages_jaunes_phone',
     'poi_pages_jaunes_website',
     'poi_pages_jaunes_reviews',
+    'poi_pages_jaunes_itinerary',
```

Comment:
I'm unable to find if this is used somewhere ?

Reply:
This is because of the system which builds poi event keys by concatenating their source and the event action… 
In https://github.com/QwantResearch/erdapfel/pull/781 I introduced the `sendPoiEvent` helper, so you have to look for a call like `Telemetry.sendPoiEvent(poi, 'itinerary');`.
It will send a `poi_pages_jaunes_itinerary` event for PagesJaunes POI and a `poi_osm_itinerary` for OSM POI.

`sendPoiEvent` was meant only as a technical simplification, but it didn't address the root of the problem, which is it's hard to find call points for these events…

Here I mimicked what was done before, because it thought it was important that we have this disctinction. But actually I'm not sure (what is sure with telemetry anyway?…)
Maybe we should just get rid of the distinction of POI sources in the event names, even for the preexisting ones. That way we could send `Telemetry.POI_ITINERARY`, that's it.

WDYT @G-Ray @amatissart ?

Reply:
Oh yes, you explained that to me earlier, however.
Ideally, I think we should use define one event per poi, and pass its type only as data, if we can afford to lose previous data...

It's not a big deal if we keep it like this, `sendPoiEvent` is indeed a simplification.

Reply:
I agree it would be cleaner to merge this `*_osm_*` and `*_pages_jaunes_*` events.  
But I am afraid that should wait for another PR, and hopefully learn more about what these events will be used for in the future.